centreon/www/class/centreonAuth.LDAP.class.php

```diff
@@ -106,7 +106,7 @@ private function getLogFlag()
      */
     public function checkPassword()
     {
-        if (empty($this->contactInfos['contact_ldap_dn'])) {
+        if (!isset($this->contactInfos['contact_ldap_dn']) || empty($this->contactInfos['contact_ldap_dn'])) {
```

Comment:
This line corrects nothing.
Check it out at this example 👉🏻  https://3v4l.org/6HMZ0

Explanation from the [php manual about empty()](https://www.php.net/manual/en/function.empty.php)
> No warning is generated if the variable does not exist.
> That means empty() is essentially the concise equivalent to !isset($var) || $var == false.